drivers/usb/usbstor/dbghlp.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * PROJECT:     ReactOS Universal Serial Bus Bulk Storage Driver
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Debug helpers
+ * COPYRIGHT:   2020 Victor Perevertkin (victor.perevertkin@reactos.org)
+ */
+
+#define NDEBUG
+#include <debug.h>
+
+#define DBGLVL_DISK (1 << 31)
+#define DBGLVL_PNP (1 << 30)
```

Comment:
Since you are using bitfield, you should (best practice) OR these with DPFLTR_MASK when you use these defines.
Alternatively you can directly use the bitfield shift number (30 and 31 directly) and this should work fine as well.
> Another easy way to use the Level parameter is to always use explicit bitfields. If you choose this method, you may wish to use a bitwise OR of the value DPFLTR_MASK with your bitfield. This value ensures that you do not accidentally use a value less than 32.

(see https://docs.microsoft.com/en-us/windows-hardware/drivers/devtest/reading-and-filtering-debugging-messages )